### DISCUSSION TITLE
Volume control not work on Iiyama G-Master GB3466WQSU

### DISCUSSION THREAD

**Discussion by leonekwolfik** [MonitorControl/MonitorControl#915] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Volume control is not working on my Iiyama G-Master GB3466WQSU monitor.

### Expected behavior

I would like to control sound volume in MonitorControl App.

### Anything else?

Does the app have some logs/raport? Can I send it to you ?

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.1
- Mac model: Mini M1 8GB
- MonitorControl version: 4.0.2 Build 6965
- Monitor(s): Iiyama G-Master GB3466WQSU
- Monitor Cable(s)/Connection(s): USB-C Thunderbolt to DisplayPort
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
This seems to be a monitor specific issue. You can try alternate apps like m1ddc to see if that app can change the volume. I'll convert this into a discussion so others with a similar display can chime in. Thank you!

**Reply by leonekwolfik** [reply]
I try m1ddc app, but I can't change the volume, I get an error:
`I2C communication failure: (iokit/?) unknown subsystem error`

```
bart@mymac m1ddc % ./m1ddc display list
1 - PL3466WQ (26CD1A76-0000-0000-331E-0104B5502278)
bart@mymac m1ddc % ./m1ddc max volume  
I2C communication failure: (iokit/?) unknown subsystem error
bart@mymac m1ddc % ./m1ddc get volume  
I2C communication failure: (iokit/?) unknown subsystem error
bart@mymac m1ddc % ./m1ddc set mute on
I2C communication failure: (iokit/?) unknown subsystem error
bart@mymac m1ddc % ./m1ddc chg volume 5
I2C communication failure: (iokit/?) unknown subsystem error
```
